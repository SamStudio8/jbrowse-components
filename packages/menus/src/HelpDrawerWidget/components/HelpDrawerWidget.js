import Typography from '@material-ui/core/Typography'
import { observer } from 'mobx-react'
import React from 'react'
import { makeStyles } from '@material-ui/core'
import Link from '@material-ui/core/Link'

const useStyles = makeStyles(theme => ({
  root: {
    margin: theme.spacing(2),
  },
  subtitle: {
    margin: theme.spacing(),
  },
}))

function Help() {
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <Typography variant="h4" align="center" color="primary">
        Help
      </Typography>
      <Typography variant="h6" align="center" className={classes.subtitle}>
        JBrowse 2.0.0-alpha.2
      </Typography>
      <Typography>
        Thanks for using this early preview version of JBrowse. If you have any
        questions, please{' '}
        <Link
          href="https://github.com/GMOD/jbrowse-components/issues/new"
          target="_blank"
          rel="noopener noreferrer"
        >
          open an issue
        </Link>{' '}
        on GitHub
      </Typography>
    </div>
  )
}

export default observer(Help)
