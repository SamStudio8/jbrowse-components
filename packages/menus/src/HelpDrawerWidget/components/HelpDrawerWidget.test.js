import React from 'react'
import { render } from 'react-testing-library'
import HelpDrawerWidget from './HelpDrawerWidget'

describe('<HelpDrawerWidget />', () => {
  it('renders', () => {
    const { container } = render(<HelpDrawerWidget />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
