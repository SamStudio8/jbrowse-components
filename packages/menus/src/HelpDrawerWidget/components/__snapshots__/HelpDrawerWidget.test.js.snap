// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelpDrawerWidget /> renders 1`] = `
<WithStyles(ForwardRef(Typography))
  variant="h6"
>
  Here's some help:
</WithStyles(ForwardRef(Typography))>
`;
